assets/src/containers/Course.js

```diff
@@ -55,7 +55,7 @@ function Course (props) {
     setSideDrawerState(open)
   }
 
-  if (error.message === 'Not Found') return (<WarningBanner>Course {courseId} has not been set up in MyLA. Contact your instructor, who can enable the visualizations by clicking on MyLA in the course navigation.</WarningBanner>)
+  if (error.message === 'Not Found' || (error.message === '')) return (<WarningBanner>Course {courseId} has not been set up in MyLA. Contact your instructor, who can enable the visualizations by clicking on MyLA in the course navigation.</WarningBanner>)
```

Comment:
I think Codacy just wants this to be cleaned up to be a little more formatted like this. I can test running this though as you have it. 

```
if (error.message === 'Not Found' || (error.message === '')) {
  return (
    <WarningBanner>
      Course {courseId} has not been set up in MyLA. Contact your instructor, who can enable the visualizations by clicking on MyLA in the course navigation.
    </WarningBanner>
  )
}
```

Reply:
Got more warnings after code reformat...